src/accordion/accordion.ts

```diff
@@ -115,13 +115,13 @@ export interface NgbPanelChangeEvent {
     <ng-template ngFor let-panel [ngForOf]="panels">
       <div class="card">
         <div role="tab" id="{{panel.id}}-header" [class]="'card-header ' + (panel.type ? 'bg-'+panel.type: type ? 'bg-'+type : '')">
-          <h5 class="mb-0">
+          <div>
```

Comment:
Don't you think that in this case we can remove this element altogether? It doesn't affect rendering, so I don't see any reasons in having it at all (that was the second part of my previous review).

On the bootstrap side they've changed example from `<h5>` → `<h2>`, but again it's just an example. Any kind of markup could be put into cards.

Reply:
agree, removed the wrapping `div` completely in the updated version.
Thanks!